modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractGraphQLCodegen.java

```diff
@@ -0,0 +1,447 @@
+
```

Comment:
Thanks for creating the abstract class. Can we copy the Apache v2 header from GraphQLServerGenerator.java and put it here?

Reply:
Yes, we can do so!

Reply:
will be done with next commit